internal/pkg/cli/project_init_test.go

```diff
@@ -299,8 +299,8 @@ func TestInitProjectOpts_Execute(t *testing.T) {
 					Create(gomock.Eq("project")).Return(nil)
 				mockProgress.EXPECT().Start(fmt.Sprintf(fmtDeployProjectStart, "project"))
 				mockDeployer.EXPECT().
-					DeployProject(&deploy.CreateProjectInput{
-						Project:    "project",
+					DeployProject(&deploy.CreateAppInput{
+						Name:       "project",
```

Comment:
```suggestion
						Name:       "app",
```

Reply:
I prefer to do this kind of change when we are renaming the `cli` package. The good thing about this approach is that we won't get confused when we refactoring the `cli` package (we'll know "app" means the previous app but not "project").